Lib/fontTools/varLib/merger.py

```diff
@@ -264,7 +264,8 @@ def merge(merger, self, lst):
 	coverageGlyphs = self.Coverage.glyphs
 	if all(v.Format == 1 for v in lst) and all(coverageGlyphs == v.Coverage.glyphs for v in lst):
 		self.Value = otBase.ValueRecord(valueFormat)
-		merger.mergeThings(self.Value, [v.Value for v in lst])
+		if valueFormat != 0:
+			merger.mergeThings(self.Value, [v.Value for v in lst])
```

Comment:
can you elaborate why this fails when valueFormat == 0? I believe it should be no-op..

Reply:
All the Values are None, and so you get a FoundNone error.

Reply:
oh got it, thanks